clouddriver-loadtest/clouddriver-loadtest.gradle

```diff
@@ -0,0 +1,27 @@
+tasks.compileGroovy.enabled = false
```

Comment:
You may want to take a review at https://github.com/spinnaker/orca/pull/1376 and roll these changes into clouddriver as well (if the PR I made looks right).